tests/scripts/branch-history

```diff
@@ -59,7 +59,7 @@ if (!chdir $mfem_dir) {
 }
 
 # Maximum number of acceptable commits in the branch
-my $branch_max_commits = 200;
+my $branch_max_commits = 500;
```

Comment:
We should probably bring this one up to the group to see if this is ok.

Reply:
There is a TODO checkbox: `Revert $branch_max_commits` to revert this before merging.